Plugin report ignores arifacts with classifiers

If the BOM has 2 artifacts that does not exist and those 2 artifacts are the same but with different classifiers, one of the artifacts is ignored from the final report.

E.g:
```
      <dependency>
        <groupId>org.kie.kogito</groupId>
        <artifactId>kogito-addons-events-decisions</artifactId>
        <version>999.0.0</version>
      </dependency>
      <dependency>
        <groupId>org.kie.kogito</groupId>
        <artifactId>kogito-addons-events-decisions</artifactId>
        <version>999.0.0</version>
        <classifier>sources</classifier>
      </dependency>
```

Report:
```
[ERROR] Failed to execute goal com.commsen.maven:bom-helper-maven-plugin:0.4.0:resolve (resolve) on project kogito-bom: The following dependencies found in <dependencyManagement> can not be resolved: 
[ERROR]  - Dependency {groupId=org.kie.kogito, artifactId=kogito-addons-events-decisions, version=999.0.0, type=jar}
```

Note that is does not report the artifact with the sources classifier.

I'm proposing that the report would show both artifacts, something like:
```
[ERROR] Failed to execute goal com.commsen.maven:bom-helper-maven-plugin:0.4.0:resolve (resolve) on project kogito-bom: The following dependencies found in <dependencyManagement> can not be resolved: 
[ERROR]  - org.kie.kogito:kogito-addons-events-decisions:jar:sources:999.0.0
[ERROR]  - org.kie.kogito:kogito-addons-events-decisions:jar:999.0.0
```

--- a/src/main/java/com/commsen/maven/plugin/bomhelper/BomResolveMojo.java
+++ b/src/main/java/com/commsen/maven/plugin/bomhelper/BomResolveMojo.java
@@ -76,7 +76,7 @@ public class BomResolveMojo extends BomHelperAbstractMojo {
 			try {
 				artifactResolver.resolveArtifact(projectBuildingRequest, coordinate);
 			} catch (ArtifactResolverException e) {
-				failedArtifacts.add(dependency.toString());
+				failedArtifacts.add(coordinate.toString());
 				logger.error("Failed to resolve artifact " + coordinate, e);
 			}
 		}

